src/templates/withDefaultContext.ts

```diff
@@ -12,7 +9,7 @@ export const withDefaultContext = (context: any = {}) => {
 
     const defaultContext: object = {
```

Comment:
Understand it is not this PR change, but typehint here actually destroys the type and casts to generic object.

Reply:
Removed